src/core/eme/types.ts

```diff
@@ -24,6 +24,20 @@ import { ICustomError } from "../../errors";
 import LoadedSessionsStore from "./utils/loaded_sessions_store";
 import PersistentSessionsStore from "./utils/persistent_sessions_store";
 
+/** Information concerning a MediaKeySession. */
+export interface IMediaKeySessionInfo {
```

Comment:
Moved here as it is now used both by the `cleanOldLoadedSessions` file and the `getSession` one.